packages/flutter/test/material/tab_bar_theme_test.dart

```diff
@@ -0,0 +1,125 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io' show Platform;
+
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+const String _tab1Text = 'tab 1';
+const String _tab2Text = 'tab 2';
+const String _tab3Text = 'tab 3';
+const String _tab4Text = 'tab 4';
+const String _tab5Text = 'tab 5';
+const String _tab6Text = 'tab 6';
+const String _tab7Text = 'tab 7';
+
+final Key _painterKey = UniqueKey();
+
+const List<Tab> _tabs = <Tab>[
```

Comment:
It's not clear why so many tabs are needed for this test?

Reply:
There actually isn't a good reason, reduced down to 3 tabs to declutter. Will update goldens now too.